modules/apps/calendar/calendar-web/src/main/resources/META-INF/resources/js/calendar_a11y.js

```diff
@@ -15,11 +15,11 @@
 AUI.add(
 	'liferay-calendar-a11y',
 	(A) => {
+		A.CalendarBase.ATTRS.tabIndex.value = 0;
+
 		let template = A.CalendarBase.HEADER_TEMPLATE;
 
 		template = template.replace('aria-role="heading"', '');
```

Comment:
@bryceosterhaus I have tested this and the accessibility test still passes when this attribute is completely removed. 

I think this pull is ready, not sure if we need someone else to look at it.

Regards.